addons/cargo/stringtable.xml

```diff
@@ -302,7 +302,7 @@
         </Key>
         <Key ID="STR_ACE_Cargo_size_edenDesc">
             <English>The cargo space required to hold this object (-1 for unloadable)</English>
-            <German>Frachtgröße zum Einladen dieses Objektes (-1 nicht einladbar)</German>
+            <German>Frachtgröße, welche zum Einladen dieses Objektes benötigt wird (-1 nicht einladbar)</German>
```

Comment:
Frachtgröße passt nicht ganz. Frachtraumgröße vllt., oder (freie) Kapazität?

Reply:
es ist die Frachtgröße, nicht die des Objektes in dessen Raum man es lädt